index.html

```diff
@@ -1287,13 +1263,27 @@
         reject <var>promise</var> with
         <code>"<a>NotSupportedError</a>"</code>, and abort these steps.
       </li>
-          <li>Let <var>adapter</var> be <code>null</code>.</li>
+      <li>
+        Let <var>adapter</var> be <code>null</code>.
+        Associate <var>adapter</var> with the current <code>Window</code>
```

Comment:
"Associate _adapter_ with" generally means the object _adapter_ references. In this case, that's always `null`. Do you mean that every `Window` object has an associated NFC adapter, initially `null`?